xbmc/utils/HevcSei.h

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (C) 2024 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "BitstreamReader.h"
+
+#include <optional>
+#include <vector>
+
+class CHevcSei
```

Comment:
A new reader might come across this class and not know what HEVC or SEI is. GPT-4 suggests some helpful documentation:

```c++
/*!
 * \brief Parses HEVC SEI messages for supplemental video information.
 *
 * The CHevcSei class is used to interpret and handle Supplemental Enhancement
 * Information (SEI) messages found in High Efficiency Video Coding (HEVC)
 * bitstreams. It is particularly useful for extracting HDR10+ metadata and
 * other types of supplemental data from HEVC encoded video streams.
 *
 * \note This class deals with SEI messages in HEVC streams and does not
 * process the video content itself.
 */
 ```

Reply:
Yeah, please add that, so we have it. Not sure someone will read the class (Bitstreamconverter) and asking himself about that, but it's good to have it.

When that comment is in - I am - for my part - perfectly fine with the PR. Thanks very much! This is a missing addition for v21, as it actually is a bugfix, cause without it  - some people cannot watch HDR10+ or DV content due to a black screen. I am one of them.

Reply:
I agree. Added it.